<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_yunTaiSubDialog</class>
 <widget class="QDialog" name="dialog_null_yunTaiSubDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox_yunTaiSubDialog_selectAll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_yunTaiSubDialog_presetList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>160</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiSubDialog_add">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiSubDialog_del">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiSubDialog_preset">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>预置点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiSubDialog_stayTime">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>停留时间</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_yunTaiSubDialog_cruiseList">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>160</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiSubDialog_startTime">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>启动时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiSubDialog_cruiseSpeed">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>巡航速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiSubDialog_cruiseNumbers">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>巡航次数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yunTaiSubDialog_startTime">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yunTaiSubDialog_cruiseSpeed">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yunTaiSubDialog_cruiseNumbers">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
